reconcile/openshift_base.py

```diff
@@ -225,7 +225,8 @@ def apply(dry_run, oc_map, cluster, namespace, resource_type, resource,
                 UnsupportedMediaTypeError):
             oc.replace(namespace, annotated.toJSON())
 
-    oc.recycle_pods(dry_run, namespace, resource_type, resource)
+    if recycle_pods:
```

Comment:
Most likely we want a log message here to tell us that we're not recycling certain pods due to the option. That way we would have a better view of what's happening.

A debug message in `oc.py` telling which pods/deployments/statefulsets/... are going to be affected may also help give visibility here...

Reply:
for terraform-resources it will be all secrets that are getting applied. so if we see an apply log, we know it will be recycled.